ansible_collections/arista/avd/plugins/modules/inventory_to_container.py

```diff
@@ -404,7 +404,7 @@ def main():
         inventory=dict(type='str', required=True),
         container_root=dict(type='str', required=True),
         configlet_dir=dict(type='str', required=False),
-        configlet_prefix=dict(type='str', required=False),
```

Comment:
unrelated change?

Reply:
yes I tried it in different branch, once confirmed I can add it to devel

Reply:
pls check only J2lint.yml

Reply:
shall I add J2Lint.yml to devel branch?

Reply:
This PR is supposed to be what will be merged into devel. So the PR should only contain the relevant changes.

Reply:
Okay ,updating it. Sorry for the inconvenience.